internal/kafka/internal/config/kafka.go

```diff
@@ -86,3 +89,22 @@ func (c *KafkaConfig) ReadFiles() error {
 func (c *KafkaConfig) Validate(env *environments.Env) error {
 	return c.SupportedInstanceTypes.Configuration.validate()
 }
+
+func (c *KafkaConfig) GetFirstAvailableSize(instanceType string) (string, error) {
+	kafkaInstanceType, err := c.SupportedInstanceTypes.Configuration.GetKafkaInstanceTypeByID(instanceType)
+	if err != nil {
+		return "", err
+	}
+	if len(kafkaInstanceType.Sizes) < 1 || kafkaInstanceType.Sizes[0].Id == "" {
+		return "", errors.New(errors.ErrorGeneral, fmt.Sprintf("Unable to get size for instance type: '%s'", instanceType))
+	}
+	return kafkaInstanceType.Sizes[0].Id, nil
+}
+
+func (c *KafkaConfig) GetKafkaInstanceSize(instanceType, sizeId string) (*KafkaInstanceSize, error) {
+	kafkaInstanceType, err := c.SupportedInstanceTypes.Configuration.GetKafkaInstanceTypeByID(instanceType)
+	if err != nil {
+		return nil, err
+	}
+	return kafkaInstanceType.GetKafkaInstanceSizeByID(instanceType, sizeId)
```

Comment:
```suggestion
	return kafkaInstanceType.GetKafkaInstanceSizeByID(sizeId)
```

We are calling a method of `KafkaInstanceType`, we don't need to pass the `instanceType`